assets/styles/pages/_list.scss

```diff
@@ -14,7 +14,7 @@
         border-radius: govuk-px-to-rem(3);
         box-shadow: 0 3px 0 #003618;
         color: #78256E;
-        padding: 8px 10px 7px;
+        padding: 8px, 10px, 7px;
```

Comment:
Is this expected? https://www.w3schools.com/cssref/pr_padding.asp

Reply:
no that was when i was testing something